url_shortener/views/__init__.py

```diff
@@ -1,3 +1,5 @@
 from .routes import setup_routes
 
 __all__ = ['setup_routes']
+
+# TODO: unit tests! Dunno what to test really
```

Comment:
Other than REGEX checks, I don't know what else can I check

Reply:
Nothing to test in this file actually.


Start with writing tests for your regexes. Check whether strings that should match according to your business logic do match actually and strings that should not --- do not match.

Secondly you have some `conditions` or input validations based on length, etc. Tests whether it works according to your business rules and that the code does exactly what you wanted it to do.

Moreover, you have storage implementation with some searches and logic built in. Test those are working correctly. For example, test that when you search for links created by one user you do not receive links created by different user.